src/main/java/lotto/Ranking.java

```diff
@@ -0,0 +1,34 @@
+package lotto;
+
+import java.util.stream.Stream;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum Ranking {
+
+	FIRST(6, 2000000000),
```

Comment:
돈과 같은 1000단위로 구분은 다음과 같이 underbar(_)를 활용하면 코드의 가독성이 좋아질 수 있어요! 😄 
```suggestion
	FIRST(6, 2_000_000_000),
```

Reply:
오 좋습니다~!^^